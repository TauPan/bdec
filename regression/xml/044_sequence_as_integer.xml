<protocol>
    <!--
      Test that we can compile non-hidden sequences with hidden children.
      -->
    <sequence name="a" value="${a:}">
        <field name="a:" length="8" type="integer" />
    </sequence>

</protocol>
