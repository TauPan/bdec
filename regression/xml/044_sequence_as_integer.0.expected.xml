<a>112</a>